openupgrade_scripts/scripts/product_expiry/17.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,13 @@
+---Models in module 'product_expiry'---
+---Fields in module 'product_expiry'---
+product_expiry / stock.quant              / expiration_date (datetime)    : NEW isrelated: related, stored
+NOTHING TO DO: ORM already filled this field by SQL being a related stored with one level deep
```

Comment:
I am not sure if it's better to fill it in pre-migration by SQL, what's your opinion @pedrobaeza ?

Reply:
It's correct this way due to the explanation given. No gain doing it manually in pre-migraiton.